src/main/resources/public/recipes.html

```diff
@@ -57,44 +57,52 @@ <h5 class="modal-title" id="cont-del-ing">Are you sure?</h5>
      </form>-->
     </div>
 </nav>
-<div style="position:relative;width:49%;float:left;padding-left:1rem;">
-    <h3><strong>Recipes:</strong>
-        <button class="btn" data-placement="right" data-toggle="tooltip" id="plus-rec"
-                style="float:right;width:3rem;height:3rem;"
-                title="Add recipe">+
-        </button>
-    </h3>
-    <hr>
-<!--    <div data-offset="0" data-spy="scroll" data-target="#recipe-list">-->
+<div class="alert-danger" id="error-message" style="display:none;text-align:center;">
+    <h3>Error</h3>
```

Comment:
When looking at this running on our Raspberry PI, I feel like this "Error" header is too much. Why not just have the error message?